packages/terra-form-select/tests/wdio/select-field-spec.js

```diff
@@ -22,6 +22,15 @@ describe('Select Field', () => {
     Terra.should.matchScreenshot('selected-option', { viewports });
   });
 
+  describe('disabled', () => {
+    it('should render a disabled Select Field', () => {
+      browser.url('/#/raw/tests/terra-form-select/form-select/disabled-select-field');
+    });
+
+    Terra.should.beAccessible({ viewports });
```

Comment:
We don't need to pass viewports to be accessible or to matchScreenshot since we are only testing one viewport size